flow/project.py

```diff
@@ -619,22 +627,23 @@ def eligible(self, job, ignore_conditions=IgnoreConditions.NONE):
         :type job:
             :class:`~signac.contrib.job.Job`
         :param ignore_conditions:
-            Specify if pre and/or post conditions check is to be ignored for eligibility check.
-            The default is :py:class:`IgnoreConditions.NONE`.
+            Specify if pre and/or post conditions check is to be ignored for
+            eligibility check.  The default is
+            :py:class:`IgnoreConditions.NONE`.
```

Comment:
```suggestion
            Specify if the pre and/or post condition checks are to be ignored
            for determining operation eligibility. The default is
            :class:`IgnoreConditions.NONE`.
```

Reply:
I made this same change in a later PR -- I'll ignore it here to avoid conflicts.